packages/wonder-blocks-dropdown/components/action-item.js

```diff
@@ -70,6 +70,11 @@ type ActionProps = {|
      * @ignore
      */
     indent: boolean,
+
+    /**
+     * Aria role to use, defaults to "menuitem".
+     */
+    role: "menuitem" | "option",
```

Comment:
Should we have a type like `DropdownItemRole` or something? This seems to be repeated often.

Reply:
I'm going to defer this for now since our documentation tool doesn't know how to expand type aliases. 😞